packages/api-file-manager-ddb/src/operations/settings/SettingsStorageOperations.ts

```diff
@@ -42,14 +43,18 @@ export class SettingsStorageOperations implements FileManagerSettingsStorageOper
 
     public async get(): Promise<FileManagerSettings | null> {
         try {
-            const settings = await this._entity.get({
-                PK: this.partitionKey,
-                SK: SORT_KEY
+            const settings = await queryOne<{ data: FileManagerSettings }>({
```

Comment:
Can you check that `query` is of the same speed as the `get`? Not sure what happens in the background.

Reply:
@brunozoric I just updated this, check my latest commit.